packages/a/appstream-glib/package.yml

```diff
@@ -0,0 +1,36 @@
+name       : appstream-glib
+version    : 0.8.2
+release    : 13
```

Comment:
`appstream-glib` was at release 13 when it was deprecated; doesn't this have to be at 14 now to undeprecate?